external-import/alienvault/src/alienvault/core.py

```diff
@@ -263,11 +263,16 @@ def run(self):
                         "Connector will not run, next run in: {0} seconds", next_run
                     )
 
-                self._sleep(delay_sec=run_interval)
             except (KeyboardInterrupt, SystemExit):
                 self._info("Connector stop")
                 exit(0)
 
+            if self.helper.connect_run_and_terminate:
+                self.helper.log_info("Connector stop")
+                exit(0)
```

Comment:
Why not use `sys.exit(0)`?

Reply:
I likely copied what the connectors were already doing. If that's the preference, its easy enough to switch.

Reply:
According to pylint (R1722: Consider using sys.exit() (consider-using-sys-exit)), it's cleaner to use `sys.exit(0)`. I am not sure if there was a specific reason to use `exit(0)` instead.

Reply:
Switched to sys.exit, commited isort changes since I was already in the import area.